src/redux/books/booksSlice.js

```diff
@@ -1,41 +1,77 @@
-import { createSlice } from '@reduxjs/toolkit';
-
-const initialState = [
-  {
-    id: 'item1',
-    title: 'The Great Gatsby',
-    author: 'John Smith',
-    category: 'Fiction',
-  },
-  {
-    id: 'item2',
-    title: 'Anna Karenina',
-    author: 'Leo Tolstoy',
-    category: 'Fiction',
-  },
-  {
-    id: 'item3',
-    title: 'The Selfish Gene',
-    author: 'Richard Dawkins',
-    category: 'Nonfiction',
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+// Add a book to the API
+export const addBookAsync = createAsyncThunk(
+  'books/addBookAsync',
+  async (newBook) => {
+    const response = await axios.post(
+      'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/XKwjcPWtaWq6gTno4u9E/books',
+      newBook,
+    );
+    return response.data;
   },
-];
-
+);
+// Remove a book from the API
+export const removeBookAsync = createAsyncThunk('books/removeBookAsync', async (bookId) => {
+  await axios.delete(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/XKwjcPWtaWq6gTno4u9E/books/${bookId}`);
+  return bookId;
+});
+const initialState = {
+  books: [],
+  isLoading: true,
+  isError: null,
+};
```

Comment:
- [ ] Well done :clap: you did a very nice job :+1: but please fix your remove functionality your remove functionality is not working so kindly fix it :slightly_smiling_face: 

![Screenshot from 2023-05-21 01-14-44](https://github.com/federicaulzurrun/Bookstore/assets/45226848/5358c8d1-7947-4562-b5d1-3f95528812b3)

- [ ] I think the problem is with your id so kindly use the correct and unique id then the remove functionality will work :+1: